x/gamm/pool-models/balancer/amm_joinpool_test.go

```diff
@@ -603,8 +626,13 @@ func (suite *KeeperTestSuite) TestCalcJoinPoolShares() {
 					require.Equal(t, sdk.NewCoins(), liquidity)
 				} else {
 					require.NoError(t, err)
-					assertExpectedSharesErrRatio(t, tc.expectShares, shares)
 					assertExpectedLiquidity(t, tc.tokensIn, liquidity)
+					err = assertExpectedSharesErrRatio(t, tc.expectShares, shares)
+					if tc.expectToleranceError {
```

Comment:
I don't see a single test case where `expectToleranceError` is true, do we need to have this conditional?